week4/w4t8.py

```diff
@@ -0,0 +1,12 @@
+n = int(input())
+dict_syn = {}
+for i in range(n):
+    s = list(map(str, input().split()))
+    dict_syn[s[0]] = s[1]
```

Comment:
как вариант можно было еще сохраянть и вторую пару в словаре, тогда заняли бы в 2 раза больше памяти, но искали бы быстрее
```suggestion
    dict_syn[word] = synonym
    dict_syn[synonym] = word
```